src/components/tailwind.preset.js

```diff
@@ -0,0 +1,146 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const defaultTheme = require('tailwindcss/defaultTheme');
+
+module.exports = {
+    theme: {
+        fontFamily: {
+            sans: [
+                'Inter',
+                ...defaultTheme.fontFamily.sans,
+            ],
+        },
+        colors: ({ colors }) => ({
+            inherit: colors.inherit,
```

Comment:
This code could be replaced with
```
  colors: ({ colors }) => ({
            ...colors, // <- here
            primary: {
                DEFAULT: '#0083FC',
                50: '#B5DBFF',
                100: '#A0D1FF',
                200: '#77BEFF',
...
```
(without need to type all colors manually).
But after this change I get warnings - the `colors` variable contains deprecated values:
```
warn - As of Tailwind CSS v2.2, `lightBlue` has been renamed to `sky`.
warn - Update your configuration file to silence this warning.

warn - As of Tailwind CSS v3.0, `warmGray` has been renamed to `stone`.
warn - Update your configuration file to silence this warning.

warn - As of Tailwind CSS v3.0, `trueGray` has been renamed to `neutral`.
warn - Update your configuration file to silence this warning.

warn - As of Tailwind CSS v3.0, `coolGray` has been renamed to `gray`.
warn - Update your configuration file to silence this warning.

warn - As of Tailwind CSS v3.0, `blueGray` has been renamed to `slate`.
warn - Update your configuration file to silence this warning.
```

Could be investigated in future;